### DISCUSSION TITLE
Yuka Lawn Sweeper manual mode

### DISCUSSION THREAD

**Discussion by kingunlim** [mikey0000/PyMammotion#59] [General]
Hello, thx for this great project!

Would it be possible to get a function where i manually can start the sweaping function like the mowing option in the addon?

best regards

**Comment by mikey0000** [comment]
Can you do it in the app? Cause if you can do it in the app I can make it work in Home assistant

6/09/2024 9:18:23 pm kingunlim ***@***.***>:

**Comment by kingunlim** [comment]
sadly i can't do it manually, its a switch i can activate when i let it mow, "collect grass oder something like that, i have the app in german, where it means "grass sammeln"
but i'm pretty sure there must be switch avaible but it is just hidden :(
if you need screenshots or anything else, just tell me.
thx!!

**Reply by kingunlim** [reply]
![image](https://github.com/user-attachments/assets/54847c91-b156-4fb5-b525-15fa45d6bd81)

**Reply by mikey0000** [reply]
Yeah the answer is a no at this point, that's part of route generation, it's not a switch in the sense you can just switch it on and off

**Reply by mikey0000** [reply]
This switch is now present in home assistant as part of route generation. I think this may be able to be turned on mid mow by modifying the mow during a mow.
